ppr-ui/src/assets/styles/base.scss

```diff
@@ -365,6 +365,9 @@ span.invalid-color {
   color: $gray9;
   font-weight: bold;
   font-size: 1rem;
+  i {
+    color: $gray9 !important;
+  }
```

Comment:
We common slot in icons with this class and manually set the colour... 
Now we can start to remove that explicit styling locally in the components.